src/templates/_includes/forms/radio.html

```diff
@@ -1,16 +1,13 @@
 {% set id = id ?? "radio#{random()}" %}
 {% set label = radioLabel ?? label ?? null %}
 
-{% set containerAttributes = containerAttributes ?? [] %}
-
-{%- if block('attr') is defined %}
-    {%- set containerAttributes = containerAttributes|merge(('<div ' ~ block('attr') ~ '>')|parseAttr, recursive=true) %}
-{% endif %}
-
 {% set inputAttributes = {
     type: 'radio',
     id: id,
     class: (class ?? [])|explodeClass,
+       class: (class ?? [])|explodeClass|merge([
+        'radio'
+    ]|filter),
```

Comment:
Looks like there was a bad merge or something here; `class` appears twice in the hash.

Reply:
@benface Thanks for pointing that out! Fixed now (647fc1b5d6fe854c2bbf95cf9a9a1591489f03e7).